src/components/scenes/SendScene2.tsx

```diff
@@ -995,9 +992,13 @@ const SendComponent = (props: Props) => {
 
           if (spendInfo.tokenId !== insufficientFunds.tokenId) {
             // Show the modal if the flip input modal is closed or save it to show when it closes later
-            const modalPromise = async () =>
-              await Airship.show(bridge => <InsufficientFeesModal bridge={bridge} coreError={insufficientFunds} navigation={navigation} wallet={coreWallet} />)
-            flipInputModalOpen.current == null ? (insufficientFeesModalPromise.current = modalPromise) : modalPromise().catch(error => showError(error))
+            if (flipInputModalRef.current != null) {
+              pendingInsufficientFees.current = insufficientFunds
+            } else {
+              await Airship.show(bridge => (
+                <InsufficientFeesModal bridge={bridge} coreError={insufficientFunds} navigation={navigation} wallet={coreWallet} />
+              )).catch(error => showError(error))
```

Comment:
Dedupe this call to InsufficientFeesModal to a `showInsufficientFeesModal(error: unknown)`